fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -310,6 +318,9 @@ public FHIRPersistenceJDBCImpl(Properties configProps, IConnectionProvider cp, F
         // TODO connect the transactionAdapter to our cache so that we can handle tx events in a non-JEE world
         this.transactionDataImpl = null;
 
+        // Always want to be testing with the new query builder
+        this.optQueryBuilderEnabled = true;
```

Comment:
There's no fhir-server-config for the unit-tests, which is where this particular constructor is used. So we set to true, because we always want to build with the new stuff. If we need to test the old stuff, we need to set up a pipeline which runs the old code. The old code gets a workout via the whole-system-search still, although that's not as comprehensive, obviously.